packages/nextjs-mf/src/loaders/fixImageLoader.ts

```diff
@@ -40,12 +40,24 @@ export async function fixImageLoader(
   const computedAssetPrefix = isServer
     ? `${Template.asString([
         'function getSSRImagePath(){',
+        //TODO: use auto public path plugin instead
```

Comment:
AutomaticPublicPath plugin needs to be rewritten now that we have new federation apis. For now i manually resolve the container over global intense.